src/excalidraw-app/index.scss

```diff
@@ -34,14 +34,6 @@
       &.ExcalidrawPlus {
         color: var(--color-promo);
       }
-      &.language {
-        padding: 0 0.625rem;
-      }
-    }
-    .dropdown-menu-group.main-menu-footer {
-      display: flex;
-      flex-direction: column;
-      row-gap: 0.5rem;
```

Comment:
we want the margin around `ChangeCanvasBackground` and `ToggleTheme` items have margin regardless of whether they're in a "footer" or not. besides, this is an excalidraw-app stylesheet.